services/email-service/handler/email.handler.ts

```diff
@@ -0,0 +1,86 @@
+import fs from 'fs';
+import FormData from 'form-data';
+import Mailgun from 'mailgun.js';
+import { Resend } from 'resend';
+import { MailtrapClient } from "mailtrap"
+import { serviceConfig } from '../configs/config';
+
+const mailgunSdk = new Mailgun(FormData);
+const mailtrapSdk = new MailtrapClient({ token: serviceConfig.mailtrap.apiKey });
+const resendSdk = new Resend(serviceConfig.resend.apiKey);
```

Comment:
can we create the instance conditionally? since I don't in one project we will use all of this provider.